requirements-dev.txt

```
@@ -1,8 +1,14 @@
 # Internationalization and Localization requirements
--e git://github.com/edx/xblock-sdk.git@v0.1.6#egg=xblock-sdk==v0.1.6
-Django==1.11.25
+-e 'git://github.com/edx/xblock-sdk.git@v0.1.6#egg=xblock-sdk==v0.1.6 ; python_version == "2.7"'
+-e 'git://github.com/edx/xblock-sdk.git@0.2.2#egg=xblock-sdk==v0.2.2 ; python_version > "2.7"'
+
+Django<1.12; python_version == '2.7'
```

@xitij2000, @shimulch, unless we want to support Django versions older than 1.11, we can use something like

```suggestion
Django~=1.11; python_version == '2.7'
```